gen/Makefile

```diff
@@ -18,18 +18,18 @@
 
 REGTOOL ?= regtool.py
```

Comment:
Can we add also the commands to generate the register list as markdown file? This command should make it: https://github.com/pulp-platform/carfield/blob/main/carfield.mk#L236

Reply:
thanks! i will include it in my next commit.